### ISSUE TITLE
Cthulhu not precompiling in 1.11. IOError: stream is closed or unusable

### ISSUE THREAD

**Issue by oxinabox** [vchuravy/Cthulhu.jl#563]
I had thought this was #343 but that is long closed (I forgot what year it was) as @aviatesk pointed out in #562 

```
(@v1.11) pkg> add Cthulhu
   Resolving package versions...
  No Changes to `~/.julia/environments/v1.11/Project.toml`
  No Changes to `~/.julia/environments/v1.11/Manifest.toml`
Precompiling project...
  ✗ Cthulhu
  0 dependencies successfully precompiled in 7 seconds. 51 already precompiled.
  1 dependency errored.
  For a report of the errors see `julia> err`. To retry use `pkg> precompile`

julia> err
PkgPrecompileError: The following 1 direct dependency failed to precompile:

Cthulhu 

Failed to precompile Cthulhu [f68482b8-f384-11e8-15f7-abe071a5a75f] to "/home/oxinabox/.julia/compiled/v1.11/Cthulhu/jl_Ee0qwa".

julia> using Cthulhu
Precompiling Cthulhu...
Info Given Cthulhu was explicitly requested, output will be shown live 
┌ Warning: Some line information is missing, type-assignment may be incomplete
└ @ Cthulhu /home/oxinabox/.julia/packages/Cthulhu/AlQim/src/codeview.jl:131
ERROR: LoadError: IOError: stream is closed or unusable
Stacktrace:
  [1] check_open(x::Base.PipeEndpoint)
    @ Base ./stream.jl:388
  [2] raw!(t::REPL.Terminals.TTYTerminal, raw::Bool)
    @ REPL.Terminals ~/.julia/juliaup/julia-1.11.0-alpha2+0.x64.linux.gnu/share/julia/stdlib/v1.11/REPL/src/Terminals.jl:138
  [3] request(term::REPL.Terminals.TTYTerminal, m::Cthulhu.CthulhuMenu; cursor::Int64, suppress_output::Bool)
    @ REPL.TerminalMenus ~/.julia/juliaup/julia-1.11.0-alpha2+0.x64.linux.gnu/share/julia/stdlib/v1.11/REPL/src/TerminalMenus/AbstractMenu.jl:241
  [4] request
    @ ~/.julia/juliaup/julia-1.11.0-alpha2+0.x64.linux.gnu/share/julia/stdlib/v1.11/REPL/src/TerminalMenus/AbstractMenu.jl:181 [inlined]
  [5] request(term::REPL.Terminals.TTYTerminal, msg::String, m::Cthulhu.CthulhuMenu; kwargs::@Kwargs{})
    @ REPL.TerminalMenus ~/.julia/juliaup/julia-1.11.0-alpha2+0.x64.linux.gnu/share/julia/stdlib/v1.11/REPL/src/TerminalMenus/AbstractMenu.jl:259
  [6] request(term::REPL.Terminals.TTYTerminal, msg::String, m::Cthulhu.CthulhuMenu)
    @ REPL.TerminalMenus ~/.julia/juliaup/julia-1.11.0-alpha2+0.x64.linux.gnu/share/julia/stdlib/v1.11/REPL/src/TerminalMenus/AbstractMenu.jl:257
  [7] _descend(term::REPL.Terminals.TTYTerminal, interp::Cthulhu.CthulhuInterpreter, curs::Cthulhu.CthulhuCursor; override::Nothing, debuginfo::Symbol, optimize::Bool, interruptexc::Bool, iswarn::Bool, hide_type_stable::Bool, verbose::Nothing, remarks::Bool, with_effects::Bool, exception_type::Bool, inline_cost::Bool, type_annotations::Bool, annotate_source::Bool, inlay_types_vscode::Bool, diagnostics_vscode::Bool, jump_always::Bool)
    @ Cthulhu ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:561
  [8] _descend(term::REPL.Terminals.TTYTerminal, interp::Cthulhu.CthulhuInterpreter, mi::Core.MethodInstance; kwargs::@Kwargs{iswarn::Bool})
    @ Cthulhu ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:791
  [9] _descend(term::REPL.Terminals.TTYTerminal, args::Any; interp::Core.Compiler.NativeInterpreter, kwargs::@Kwargs{iswarn::Bool})
    @ Cthulhu ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:807
 [10] __descend_with_error_handling(args::Any; terminal::Any, kwargs...)
    @ Cthulhu ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:222
 [11] _descend_with_error_handling(f::Any, argtypes::Any; kwargs::@Kwargs{iswarn::Bool, terminal::REPL.Terminals.TTYTerminal})
    @ Cthulhu ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:211
 [12] descend_code_typed(::Any, ::Vararg{Any}; kwargs::@Kwargs{terminal::REPL.Terminals.TTYTerminal})
    @ Cthulhu ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:169
 [13] macro expansion
    @ ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:883 [inlined]
 [14] macro expansion
    @ ~/.julia/packages/PrecompileTools/L8A3n/src/workloads.jl:78 [inlined]
 [15] macro expansion
    @ ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:882 [inlined]
 [16] macro expansion
    @ ~/.julia/packages/PrecompileTools/L8A3n/src/workloads.jl:140 [inlined]
 [17] top-level scope
    @ ~/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:878
 [18] include
    @ ./Base.jl:558 [inlined]
 [19] include_package_for_output(pkg::Base.PkgId, input::String, depot_path::Vector{String}, dl_load_path::Vector{String}, load_path::Vector{String}, concrete_deps::Vector{Pair{Base.PkgId, UInt128}}, source::Nothing)
    @ Base ./loading.jl:2679
 [20] top-level scope
    @ stdin:4
in expression starting at /home/oxinabox/.julia/packages/Cthulhu/AlQim/src/Cthulhu.jl:1
in expression starting at stdin:4
  ✗ Cthulhu
  0 dependencies successfully precompiled in 7 seconds. 18 already precompiled.

ERROR: The following 1 direct dependency failed to precompile:

Cthulhu 

Failed to precompile Cthulhu [f68482b8-f384-11e8-15f7-abe071a5a75f] to "/home/oxinabox/.julia/compiled/v1.11/Cthulhu/jl_6xbjy8".
Stacktrace:
  [1] error(s::String)
    @ Base ./error.jl:35
  [2] precompilepkgs(pkgs::Vector{…}; internal_call::Bool, strict::Bool, warn_loaded::Bool, timing::Bool, _from_loading::Bool, configs::Pair{…}, io::Base.TTY, fancyprint::Bool)
    @ Base.Precompilation ./precompilation.jl:935
  [3] _require(pkg::Base.PkgId, env::String)
    @ Base ./loading.jl:2336
  [4] __require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base ./loading.jl:2175
  [5] #invoke_in_world#3
    @ ./essentials.jl:1064 [inlined]
  [6] invoke_in_world
    @ ./essentials.jl:1061 [inlined]
  [7] _require_prelocked(uuidkey::Base.PkgId, env::String)
    @ Base ./loading.jl:2166
  [8] macro expansion
    @ ./loading.jl:2106 [inlined]
  [9] macro expansion
    @ ./lock.jl:273 [inlined]
 [10] __require(into::Module, mod::Symbol)
    @ Base ./loading.jl:2063
 [11] #invoke_in_world#3
    @ ./essentials.jl:1064 [inlined]
 [12] invoke_in_world
    @ ./essentials.jl:1061 [inlined]
 [13] require(into::Module, mod::Symbol)
    @ Base ./loading.jl:2056
Some type information was truncated. Use `show(err)` to see complete types.

julia> versioninfo()
Julia Version 1.11.0-alpha2
Commit 9dfd28ab751 (2024-03-18 20:35 UTC)
Build Info:
  Official https://julialang.org/ release
Platform Info:
  OS: Linux (x86_64-linux-gnu)
  CPU: 20 × 12th Gen Intel(R) Core(TM) i7-12700H
  WORD_SIZE: 64
  LLVM: libLLVM-16.0.6 (ORCJIT, alderlake)
Threads: 1 default, 0 interactive, 1 GC (on 20 virtual cores)
Environment:
  JULIA_PKG_SERVER = 
  JULIA_PKG_USE_CLI_GIT = true

```

**Comment by aviatesk** [comment]
Ok, now I think this is dup of https://github.com/JuliaDebug/Cthulhu.jl/issues/541 , which was fixed by https://github.com/JuliaLang/julia/pull/53822 and will be included in the next 1.11 release.

**Comment by oxinabox** [comment]
ah ok, i think i just going the error number wrong
